public/app/core/utils/fetch.ts

```diff
@@ -1,3 +1,4 @@
+import { Blob } from 'buffer';
```

Comment:
Ah ok this seems to be the problem. This is buffer from the node API which we (probably) don't want to use. We want to use the browser Blob (which is available globally) or use the polyfill in Jest. With this import you created a "native" Blob in the test but then were comparing it with the node Blob imported here (as that shadowed the global one).

Reply:
honestly I wasn't able to make it work